src/main/java/pcl/lc/irc/hooks/EightBall.java

```diff
@@ -23,19 +23,17 @@ protected void initHook() {
 		local_command = new Command("eightball", 0) {
 			@Override
 			public void onExecuteSuccess(Command command, String nick, String target, GenericMessageEvent event, String params) {
-				if (params.length() > 6) {
-					if (params.matches(".*\\?$")) {
-						ArrayList<String> messages = new ArrayList<>();
-						messages.add("Signs point to yes");
-						messages.add("Without a doubt");
-						messages.add("Reply hazy, try again");
-						messages.add("Ask again later");
-						messages.add("My reply is no");
-						messages.add("Outlook not so good");
-						messages.add("[ The Bowling ball doesn't answer ]");
-						Helper.sendMessage(target, messages.get(Helper.getRandomInt(0, messages.size() - 1)), nick);
-						return;
-					}
+				if ((params.length() > 6 && params.matches(".*\\?$")) || params.equals("^")) {
```

Comment:
Worth noting -- I'm not overly familiar with Java, so I'm not sure if `.equals()` is the right method, I just remember that `==` won't work.